src/Teams/appsettings.Development.json

```diff
@@ -1,6 +1,6 @@
 {
   "ConnectionStrings": {
-    "DefaultConnection": "Server=(local);Database=TeamCapacity;MultipleActiveResultSets=true;Trusted_Connection=True;"
+    "DefaultConnection": "Server=(localdb)\\MSSQLLocalDB;Database=TeamCapacity;MultipleActiveResultSets=true;Trusted_Connection=True;"
```

Comment:
Please remember to remove this from the changes when you commit.

Right now please change to previous connection string.